test/runtests.jl

```diff
@@ -477,5 +478,22 @@ using Compat
             @test_broken isready(bottom_dc3) == true
             @test_broken fetch(bottom_dc3) == 14
         end
+
+        @testset "DeferredFuture parallel serialization" begin
+            pnums = addprocs(2)
+            @everywhere using Dispatcher
+
+            try
+                ctx = DispatchContext()
+                exec = ParallelExecutor()
+
+                op = Op(()->3)
+                add!(ctx, op)
+
+                results = run!(exec, ctx)
```

Comment:
You mean like this?

```using DeferredFutures
addprocs(2)
@everywhere using DeferredFutures

df = DeferredFuture()

@spawn begin
    df2 = DeferredFuture()
    put!(df2, 3)
    put!(df, df2)
end

@assert fetch(fetch(df)) == 3```

This seems to work fine. I've tried recreating the error without using Dispatcher but haven't been able to yet.